raft/server.cc

```diff
@@ -1301,6 +1310,7 @@ future<> server_impl::abort() {
 }
 
 future<> server_impl::set_configuration(config_member_set c_new, seastar::abort_source* as) {
+    check_not_aborted();
```

Comment:
I do not think this is our public API any longer. `modify_config` is. We should hide this in private. @kostja ? Also we already have a check for `_abort` in  `wait_for_entry()` which this function is calling. The work it does before doing so is insignificant.

Reply:
We use it in tests, so we can't make it private. We could try to switch tests to modify_config(), I guess.

Reply:
removed `check_not_aborted` from it, created an [issue](https://github.com/scylladb/scylladb/issues/11447) about excluding if from the public api